developer-docs-site/docs/nodes/ait/index.md

```diff
@@ -0,0 +1,159 @@
+      </div>
+    </div>
+  </div>
+  <div class="col" >
+    <div class="card h-100">
+     <h3 class="card-header">Step 2</h3>
+      <div class="card-body d-flex flex-column">
+      <a href="#install-the-nodes-for-ait-2" class="card-title card-link stretched-link"> <h2>Install the nodes</h2></a>
+        <p class="card-text">Ready to dive in? Follow these guides to install and set up the nodes for the AIT-2.</p>     
+      </div>
+    </div>
+  </div>
+  <div class="col" >
+    <div class="card h-100">
+     <h3 class="card-header">Step 3</h3>
+      <div class="card-body d-flex flex-column">
+      <a href="#staking-for-the-ait-2" class="card-title card-link stretched-link"> <h2>Stake on the AIT-2</h2></a>
+        <p class="card-text">Selected for the AIT-2 Validator set? Follow this guide and stake your Aptos coins.</p>     
+      </div>
+    </div>
+  </div>
+</div>
+</div>
+
+## AIT-2 program
+
+In order to participate in the incentivized testnet, participants must demonstrate the ability to configure and deploy a node, as well as pass the sanctions screening requirements.
+
+<div class="docs-card-container">
+<div class="step">
+    <div>
+        <div class="circle">1</div>
+    </div>
+    <div>
+        <div class="step-title">Join the Aptos Discord community</div>
+        <div class="step-caption">Visit the <a class="reference external" href="https://community.aptoslabs.com/">Aptos Community page</a> and join.  </div>
+    </div>
+</div>
+<div class="step">
+    <div>
+        <div class="circle">2</div>
+    </div>
+    <div>
+        <div class="step-title">Read the Node Requirements</div>
+        <div class="step-caption">Before you proceed, make sure that you satisfy the <a href="node-requirements">Node Requirements</a>.  </div>
+    </div>
+</div>
+<div class="step">
+    <div>
+        <div class="circle">3</div>
+    </div>
+    <div>
+        <div class="step-title">Follow the instructions to install and deploy both a Validator node and a FullNode in the test mode.</div>
+        <div class="step-caption">Follow the detailed node installation steps in the section: <a href="#install-the-nodes-for-ait-2">Install the nodes for AIT-2</a>. <strong>Make sure to set your node in the Test mode.</strong> Instructions are provided in the node installation sections. </div>
+    </div>
+</div>
+<div class="step">
+    <div>
+        <div class="circle">4</div>
+    </div>
+    <div>
+        <div class="step-title">Provide your node details to the Aptos Discord community</div>
+        <div class="step-caption">Navigate to the <a href="https://community.aptoslabs.com/">Incentivized Testnet registration page</a> and enter your Validator's information. Provide your account address, your public keys, and your Validator's network addresses. Optionally, you can also provide your FullNode details. </div>
```

Comment:
Do we want to provide more details on how to find the pubkey so people don't accidentally submit their private key instead? This was one of the major problems in AIT1.